### DISCUSSION TITLE
Is it possible to avoid seleniumbase from closing the browser when the python program ends?

### DISCUSSION THREAD

**Discussion by DanBaires** [seleniumbase/SeleniumBase#2362] [General]
Hello, I'm new at Selenium base (new at browser automation also...). I need to do some automation, but at certain point, I need to allow a "real person" to continue form the point where the program finish... The person will have to work for a long time, so I can't just add a sleep sentence. Thank you so much in advance!

**Comment by mdmintz** [comment]
Just drop in a `breakpoint()` anywhere in your code to pause it. Type `c` + `Enter/Return` to continue from the breakpoint when you're done performing manual actions.

https://seleniumbase.com/debugging-with-the-new-pdbp-debugger/

**Comment by DanBaires** [comment]
Thank you very much!!!
